get-resource.sh

```diff
@@ -19,9 +19,9 @@ cd /shared/html/images
 # we don't need to download it
 if [[ -e /var/tmp/$FILENAME.initramfs && \
       -e /var/tmp/$FILENAME.kernel ]] ; then
-    mv /var/tmp/$FILENAME.initramfs $FILENAME.initramfs
-    mv /var/tmp/$FILENAME.kernel $FILENAME.kernel
-    mv /var/tmp/ipa-ramdisk-pkgs.info ipa-ramdisk-pkgs.info
+    cp /var/tmp/$FILENAME.initramfs $FILENAME.initramfs
+    cp /var/tmp/$FILENAME.kernel $FILENAME.kernel
+    cp /var/tmp/ipa-ramdisk-pkgs.info ipa-ramdisk-pkgs.info
     exit 0
```

Comment:
I'm not sure this will have a different effect, in general when working with systems where SELinux is enabled, it's recommended to use the -Z option.
In this case, I suggest to keep mv but add -Z, or use restorecon after moving the files.
If we want to use cp, we should also use -Z (or use restorecon), but we also need to delete the original files afterwards, which means one step more than using mv.

Reply:
My first attempt was to use "mv -Z" but it didn't work(not sure why), so I switched it to use cp instead. 
See below, the "c223,c733" context is causing the problem, as it changes if the container is restarted.

I didn't think we would save any space by removing the file afterwards as it would still be part of the file system layers but looks like I was wrong, I'll remove it 


```
[root@localhost /]# ls -lZ  /var/tmp/ironic-python-agent.*
-rw-r--r--. 1 root root system_u:object_r:container_file_t:s0:c223,c733 535998037 Jun 29 09:31 /var/tmp/ironic-python-agent.initramfs
-rwxr-xr-x. 1 root root system_u:object_r:container_file_t:s0:c223,c733   8110944 Jun 29 09:31 /var/tmp/ironic-python-agent.kernel

[root@localhost /]# mv -Z /var/tmp/ironic-python-agent.kernel  shared/html/images/ironic-python-agent.kernel
[root@localhost /]# cp /var/tmp/ironic-python-agent.initramfs shared/html/images/ironic-python-agent.initramfs

[root@localhost /]#  ls -lZ shared/html/images/ironic-python-agent.* /var/tmp/ironic-python-agent.*               
-rw-r--r--. 1 root root system_u:object_r:container_file_t:s0:c223,c733 535998037 Jun 29 09:31 /var/tmp/ironic-python-agent.initramfs
-rw-r--r--. 1 root root system_u:object_r:container_file_t:s0           535998037 Jun 29 10:41 shared/html/images/ironic-python-agent.initramfs
-rwxr-xr-x. 1 root root system_u:object_r:container_file_t:s0:c223,c733   8110944 Jun 29 09:31 shared/html/images/ironic-python-agent.kernel

```

Reply:
interesting, root has access to all MCS ranges so that shouldn't be a problem.
I guess we could control the security levels assigned to the objects (containers or pods, or even the volume itself) in kubernetes, but this is probably too complex for what we need in this case.
Anyway, this looks good to me, thanks! :)